### ISSUE TITLE
doesn't play nice with pyodbc

### ISSUE THREAD

**Issue by dmvianna** [apple/turicreate#93]
Tested in `Jupyter`

```python
import pyodbc
import turicreate as tc

with open('turi_auth.txt', 'r') as auth:
    conn = pyodbc.connect(auth.read())

data = tc.SFrame.from_sql(conn, "SELECT * FROM mytable", dbapi_module=pyodbc)
print data
```
Error message
```python
ERROR:root:An unexpected error occurred while tokenizing input
The following traceback may be corrupted or invalid
The error message is: ('EOF in multi-line string', (1, 0))

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-7745e517349c> in <module>()
     30 
     31 WHERE v.site_street_number != ''
---> 32 """, dbapi_module=pyodbc)
     33 data

/home/user/.venvs/env/local/lib/python2.7/site-packages/turicreate/data_structures/sframe.pyc in from_sql(cls, conn, sql_statement, params, type_inference_rows, dbapi_module, column_type_hints, cursor_arraysize)
   1814             sb = SFrameBuilder(result_types, column_names=result_names)
   1815 
-> 1816         sb.append_multiple(_force_cast_sql_types(temp_vals, result_types, cols_to_force_cast))
   1817         rows = c.fetchmany()
   1818         while len(rows) > 0:

/home/user/.venvs/env/local/lib/python2.7/site-packages/turicreate/data_structures/sframe_builder.pyc in append_multiple(self, data, segment)
    156         if hasattr(data, '__len__'):
    157             if len(data) <= self._block_size:
--> 158                 self._builder.append_multiple(data, segment)
    159                 return
    160 

turicreate/cython/cy_sframe_builder.pyx in turicreate.cython.cy_sframe_builder.UnitySFrameBuilderProxy.append_multiple()

turicreate/cython/cy_sframe_builder.pyx in turicreate.cython.cy_sframe_builder.UnitySFrameBuilderProxy.append_multiple()

turicreate/cython/cy_flexible_type.pyx in turicreate.cython.cy_flexible_type.flex_list_from_iterable()

turicreate/cython/cy_flexible_type.pyx in turicreate.cython.cy_flexible_type.common_typed_flex_list_from_iterable()

turicreate/cython/cy_flexible_type.pyx in turicreate.cython.cy_flexible_type.tr_buffer_to_flex_list()

TypeError: Could not convert python object with type Row to flex list.
```

**Comment by dmvianna** [comment]
And before you ask: yes, the `turi_auth.txt` file is fine and works trivially with `pandas`

```python
import pandas as pd
pd.read_sql_query("""select * from mytable""", conn)
```

**Comment by srikris** [comment]
Thanks for pointing out! This looks like a type mis-match. Hopefully, its a simple fix. We will look into it.

**Comment by srikris** [comment]
Fixed in #109
